server/app/controllers/api/v1/experiences_controller.rb

```diff
@@ -1,11 +1,13 @@
 require 'uri'
+require 'pry'
 class Api::V1::ExperiencesController < ApplicationController
+  skip_before_action :authorize_request, only: [:show, :index]
   before_action :set_api_user
   before_action :set_api_user_experience, only: [:show, :update, :destroy]
 
   #GET /api_users/:api_user_id/experiences
   def index
-    @experiences = @api_user.experiences.includes(:programminglanguage, :meetinginterval)  if params[:api_user_id]
+    @experiences = @api_user.experiences.includes(:programminglanguage, :meetinginterval)  # if params[:api_user_id]
```

Comment:
Do we intend to keep the commented-out code? If no, remove it.

Reply:
no it is not needed anymore, I have removed it.